### PR TITLE
Support PidLogicalAssembly and PidLogicalAssemblyOwnsPidGraphic - Used in Smart Plant Instrumentation (SPI) Connector

### PR DESCRIPTION
**Pull request by jchick-bentley**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -541,13 +541,10 @@
       }
```
Comment (naveedkhan8067) [inline package-lock.json:541]: Can you please revert the changes in `package-lock.json`. Are they intentional?

File: SchemaInventory.json [hunk 2]
```diff
@@ -1208,6 +1208,20 @@
...
       "dynamic": "No",
       "approved": "Yes"
+    },
+    {
+      "name": "ProcessPidGraphical",
+      "path": "Domains\\4-Application\\Plant\\Released\\ProcessPidGraphical.01.00.03.ecschema.xml",
+      "released": true,
+      "version": "01.00.03",
+      "comment": "",
+      "verifier": "",
```
Comment (naveedkhan8067) [inline SchemaInventory.json:1218]: Can you please add a `verifier`? and then make the `verified` and `approved` fields to `yes`.

File: SchemaInventory.json [hunk 3]
```diff
@@ -1173,7 +1173,7 @@
       "name": "ProcessPidGraphical",
       "path": "Domains\\4-Application\\Plant\\ProcessPidGraphical.ecschema.xml",
       "released": false,
-      "version": "01.00.02",
+      "version": "01.00.03",
```
Comment (naveedkhan8067) [inline SchemaInventory.json:1176]: It will make the working copy and the latest released one having same versions. Working copy usually represents the next version e.g `01.00.04`.

File: package-lock.json (hunk 1 above)
Comment (jchick-bentley) [reply package-lock.json:541]: @naveedkhan8067 Yes, I can revert.  I got a "severe/critical" warning on npm install and followed the prompt to do "npm audit fix" which eliminated the highest severity vulnerability and some of the moderate ones.  They are intentional in the sense that I was aware of the changes and deliberately staged, committed, and pushed them as I'm accustomed to do in other repos.  LMK and I'll revert promptly.

File: SchemaInventory.json (hunk 2 above)
Comment (jchick-bentley) [reply SchemaInventory.json:1218]: @diegoalexdiaz Can you please be the "verifier" for this PR?  If you agree, I'll update the SchemaInventory.json with your name and a "yes" in the verified field.

**Comment by jchick-bentley** [comment]
@naveedkhan8067 this is ready for a review - all your comments are addressed. Thank you.

File: package-lock.json (hunk 1 above)
Comment (naveedkhan8067) [reply package-lock.json]: Thanks @jchick-bentley. I will take a look to those warnings.

**Review by naveedkhan8067 — APPROVED** [review]

**Comment by naveedkhan8067** [comment]
> @naveedkhan8067 this is ready for a review - all your comments are addressed. Thank you.

Thanks @jchick-bentley!

**Review by ColinKerr — APPROVED** [review]
